README.md

```diff
@@ -38,13 +38,15 @@ steps:
     settings:
       mode: upgrade
       chart: ./
+      lint: true
```

Comment:
I would simply set this example as `lint: false` and remove the Note below. To me the note below is confusing and is trying to explain something after the fact, so people might not even read it.

Reply:
I meant changing the value and removing the Note altogether